samples/v1.3/Scenarios/CalendarReminder.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
These new cards need a new name (like "Calendar Reminder with labels")

Or else the site gets very confused:
https://adaptivecardsci.z5.web.core.windows.net/pr/4561/samples/CalendarReminder.html